flyteadmin_config.yaml

```diff
@@ -60,6 +60,7 @@ flyteadmin:
   metadataStoragePrefix:
     - "metadata"
     - "admin"
+  offloadWorkflowClosureToStorage: false
```

Comment:
Can we also add this flag in the values file with a comment on it usage

Reply:
Also a general question , since this is an application wide setting, 
Should we also be able to do this at the creation of execution level through a request spec